site/_posts/2021-08-18-annnoucing-fusionauth-1.30.md

```diff
@@ -0,0 +1,89 @@
+
+To solve this, we have created the possiblity of opening an API that allows a JWT to be created with a payload defined by the API called.  This would require an API key, so the caller must be privledged.  
+
+Here is an example of the endpoint at work: 
+
+Endpoint: /api/jwt
+
+```
+
+{
+  "claims": {
+     "roles": [ "awesome" ],
+     "id": 42
+   }
+}
+```
+
+Using this ability, a new user on your app can start interacting with the app as a unique user, before needing to create an account. Then, once they do create an account, their unique Id is translated onto their profile. 
+
+## Threat Detection Feature
+
+Flagging and responding to suspicious behaviour is a huge advantage in any cybersecurity product, and we wanted to bring our customers as many advantages as we could in this arena. So, we created a large feature that has best-practice funtionality built in to deal with any sort of bizarre behaviours around user creation, user updates, and IP location and fingerprinting. 
+
+While the entirety of this feature is too large to cover here, we want to call out a few notable aspects:
+
+Rate limiting per user for specific requests such as:
+ - Failed login
+ - Forgot password
+ - Send email verification
+ - Send passwordless
+ - Send registration verification
+ - Send two-factor
+ 
+Capcha - Uses additonal (re)captcha to guard against brute force attacks. 
+- Adds a (re)captcha for login pages which use an email/password combination
+- Adds support for Google ReCaptcha v2, Google ReCaptcha v3, HCaptcha and Hcaptcha Enterprise
+
+IP Location Services
+- Includes the IP address of the geographic location where a Forgot/Reset Password email was created, so the recipient can identify if the location is genuinely their request or not. 
+- Flags suspicious IP adddresses on login and sends an email to notify the user of a new login with the IP's approximate location
+- Calculates 'impossible travel' to see if a user could genuinely login in different locations around the globe in a reasonable time frame. (This does not provide support if your user is a quantum being...)
+- With unexpected IP addresses, sends a user an email to notify them of a new login with an approximate location of the IP address. 
+
+To learn more, you can take a look at our [release notes](/docs/v1/tech/release-notes/#version-1-30-0). 
+
+## Webhook Events
+
+We've added 38 webhook events to help you moniter what is going on with your app.  These webhooks cover an entire breadth of createing, updated, and deleting a user, as well as the user registration and password reset lifecycle. Lastly, we've added some webhook events to moniter any kickstart files you may create, as well as moniter JWT behavior with refresh tokens and public/private keypairs are updated. 
+
+## The rest of it
+
+Some of the other enhancements and fixes included in this release:
```

Comment:
If these are sentences (and it seems they are) please capitalize the first letter and add a period at the end.

Reply:
If they aren't, capitalize the first letter and no period.

Reply:
But have it be consistent.